bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -733,6 +733,8 @@ lv_display_t *bsp_display_start(void)
             .sw_rotate = true,
         }
     };
+    cfg.lvgl_port_cfg.task_affinity = CONFIG_BSP_DISPLAY_LVGL_TASK_AFFINITY;
+    cfg.lvgl_port_cfg.task_priority = CONFIG_BSP_DISPLAY_LVGL_TASK_PRIORITY;
```

Comment:
Generally, we prefer setting these parameters from run-time (instead of build-time with Kconfig).

Does `bsp_display_start_with_config()` function meet your requirements?